tests/parse/test_parse_hpo_mappings.py

```diff
@@ -8,9 +8,9 @@ def test_parse_hpo_to_genes(hpo_disease_handle):
     hpo_2_genes = parse_hpo_to_genes(hpo_disease_handle)
 
     # THEN the iterator returned should contain hpo_id and hgnc_symbol
-    for item in hpo_2_genes:
-        assert item["hpo_id"]
-        assert item["hgnc_symbol"]
+    for hpo_id, gene_list in hpo_2_genes.items():
```

Comment:
While I dont mind this version, I also dont quite see what was wrong with the previous?